__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index')
+const request = require('supertest');
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe('/api/topics', () => {
+    test('200: GET /api/topics - responds with an array of topic objects', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            const { topics } = body;
+            
+            expect(topics.length).toBe(3);
+            
+            topics.forEach((topic) => {
+                expect(typeof topic.description).toBe('string');
```

Comment:
a better jest matcher for testing objects, especially as you'll start testing bigger ones in the next few tickets, is .toMatchObject or .objectContaining. I'd suggest having a go at using them on future tickets